solutions.txt

```diff
@@ -136,9 +125,12 @@ const nums = [100, 5, 23, 15, 21, 72, 9, 45, 66, 7, 81, 90];
 
 // Complete Exercise 11 below...
 
-
+let odds = [5, 23, 15, 21, 9, 45, 7, 81];
 
 console.log('Exercise 11 Result:\n', odds);
+nums.forEach(element => console.log(element));
+//I honestly wasnt sure on how to iterate, I found this document on google that walked me through a method but still was very foregin to me. I also had no clue on how to iterate through the provided nums array add each odd number to a new array named odds. Im pretty sure that there is a way to properly do that but this is the only way that made sense to me. I am curious to know what that way is because I couldnt seem to find it googling.
```

Comment:
the big idea with iterators aka array methods is that once you're inside the callback (the arrow function inside the iterator function), you can forget about everything and just write out what you need to do for ONE of the elements.

in this case, the `forEach()` will take care of iterating through everything, and inside the callback you just need to say what it is that we want to do to each element in the array. here, we want to just `console.log()` the element. 

what you did looks exactly right, but understanding it can be a separate step. Let me know if this is still unclear and we can continue to talk through it!